asb-native/src/main/java/org/ballerinalang/asb/AsbMessageUtils.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.asb;
+
+import org.ballerinalang.jvm.JSONParser;
+import org.ballerinalang.jvm.XMLFactory;
+import org.ballerinalang.jvm.api.values.BArray;
+import org.ballerinalang.jvm.api.BStringUtils;
+
+import java.io.UnsupportedEncodingException;
+import java.nio.charset.StandardCharsets;
+
+/**
+ * Util class for Asb Message handling.
+ */
+public class AsbMessageUtils {
+    public static Object getTextContent(BArray messageContent) {
+        byte[] messageCont = messageContent.getBytes();
+        try {
+            return BStringUtils.fromString(new String(messageCont, StandardCharsets.UTF_8.name()));
+        } catch (UnsupportedEncodingException exception) {
+            return AsbUtils.returnErrorValue(AsbConstants.TEXT_CONTENT_ERROR
+                    + exception.getMessage());
```

Comment:
Shall we move this to the previous line?
(For all the similiar return statements in this file)
We can break a statement if the number of characters exceed 120

Reply:
Resolved. Thank you